JSDemos/Demos/HtmlEditor/Tables/description.md

```diff
@@ -0,0 +1,9 @@
+DevExtreme **HtmlEditor** allows users to create and manage tables. To enable this functionality, add the following [items](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Configuration/toolbar/items/) to the [toolbar](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Configuration/toolbar/):
+
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#inserttable" target="_blank">insertTable</a>
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertrowabove" target="_blank">insertRowAbove</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertrowbelow" target="_blank">insertRowBelow</a>
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnright" target="_blank">insertColumnRight</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnleft" target="_blank">insertColumnLeft</a>
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletetable" target="_blank">deleteTable</a>
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deleterow" target="_blank">deleteRow</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletecolumn" target="_blank">deleteColumn</a>
+
+When users click a toolbar item, a method with the same name is called. These methods belong to the Table module of Quill. You can use the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method to access this module and call its methods at any point in your application. Refer to the following documentation file to see the full list of the Table methods with examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md">DevExtreme Quill modules: Table</a>.
```

Comment:
There's a lot of repetition of "modules" and "methods" so here's my compressed version - I think it ultimately says the same thing: 

```suggestion
You can initiate the same actions from code. Use the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method to access the Table module. This module contains methods whose names match the options listed above. Refer to the following topic for the complete list and examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md">DevExtreme Quill modules: Table</a>.
```